Frontend/src/pages/PublishArticle.tsx

```
@@ -1,5 +1,18 @@
+<<<<<<< Updated upstream
 import { X } from 'lucide-react'
 import React, { useRef, useState } from 'react'
+=======
+import BounceLoader from "@/Components/BounchLoader";
+import { useFetchQuery } from "@/hooks/useFetchQuery";
+import { useHttp } from "@/hooks/useHttp";
+import { IUser } from "@/Interfaces/AuthInterfaces";
+import useTokenStore from "@/store/TokenStore";
+import { useQuery } from "@tanstack/react-query";
+import { X } from "lucide-react";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'X'. Consider to delete it or rename it.

'X' is defined here:


(lint/suspicious/noRedeclare)